src/improvements/justfile

```diff
@@ -171,16 +171,97 @@ simulate-all-templates:
         exit 1
     fi
 
-simulate-verify-task TASK="":
+# Simulate a task locally and against Tenderly.
```

Comment:
Why do we need a new base contracts version?

Reply:
We don't, really. Reverted the update.